enaml/compile_all.py

```diff
@@ -0,0 +1,88 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2018, Nucleic Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#------------------------------------------------------------------------------
+""" Command-line tool to compile .py and .enaml files.
+
+"""
+import os
+import sys
+import compileall
+
+from enaml.compat import IS_PY3
+from enaml.core.import_hooks import EnamlImporter, make_file_info
+
+
+# We redefine this so create a local reference
+compile_py_file = compileall.compile_file
+
+
+def compile_enaml_file(fullname, ddir=None, force=0, rx=None, quiet=0,
+                       *args, **kwargs):
+    """Byte-compile one file using the EnamlImporter.
+    
+    """
+    fullname = os.path.abspath(fullname)
+    importer = EnamlImporter(make_file_info(fullname))
+    if not quiet:
+        print('Compiling {}...'.format(fullname))
+    try:
+        if force:
+            importer.compile_code()
+        else:
+            importer.get_code()
+        return True if IS_PY3 else 1
+    except Exception as e:
+        if quiet:
```

Comment:
Not sure I understand this conditional.

Reply:
Just keeps the return value consistent with what cpython has.

https://github.com/python/cpython/blob/2.7/Lib/compileall.py#L72

and

https://github.com/python/cpython/blob/3.6/Lib/compileall.py#L109

Reply:
Make sense I was thinking "verbose" not "quiet" hence my confusion.